### ISSUE TITLE
Exchange retriever halts on invalid unicode

### ISSUE THREAD

**Issue by scroix** [museumsvictoria/nodel-recipes#23]
Had a funny one where somebody used an unusual unicode apostrophe `’` in place of a regular `'` apostrophe when entering an event on the exchange calendar.

```
12-01 19:04:14.59 java.lang.RuntimeException: java.lang.RuntimeException: Traceback (most recent call last):
12-01 19:04:14.59   File "C:\Nodel\nodel-WEB\nodel-dashboard\.\nodes\MV Gallery Calendar - Exchange Retriever\script.py", line 146, in local_action_PollItems
12-01 19:04:14.59     rawBookings = query_ews(now, now.plusDays(7))
12-01 19:04:14.59   File "C:\Nodel\nodel-WEB\nodel-dashboard\.\nodes\MV Gallery Calendar - Exchange Retriever\script.py", line 235, in query_ews
12-01 19:04:14.59     items = parse_query_response(response, warnings)
12-01 19:04:14.59   File "C:\Nodel\nodel-WEB\nodel-dashboard\.\nodes\MV Gallery Calendar - Exchange Retriever\script.py", line 240, in parse_query_response
12-01 19:04:14.59     root = ET.fromstring(responseXML)
12-01 19:04:14.59   File "C:\Nodel\nodel-JARS\nodelhost-dev-2.1.1-rev293.jar\Lib\xml\etree\ElementTree.py", line 976, in XML
12-01 19:04:14.59   File "C:\Nodel\nodel-JARS\nodelhost-dev-2.1.1-rev293.jar\Lib\xml\etree\ElementTree.py", line 1258, in feed
12-01 19:04:14.59   File "C:\Nodel\nodel-JARS\nodelhost-dev-2.1.1-rev293.jar\Lib\xml\parsers\expat.py", line 193, in Parse
12-01 19:04:14.59 UnicodeEncodeError: 'ascii' codec can't encode characters in position 13306-13308: ordinal not in range(128)
```

It otherwise looked okay! But caused the **entire node to fail**.

<img width="350" alt="screen shot 2017-12-01 at 7 14 14 pm" src="https://user-images.githubusercontent.com/9277107/33475029-f6e416c8-d6d0-11e7-9519-b43f7574d693.png">

**Comment by stefmcd** [comment]
I found that location information had been copied and pasted from an external document (Google Drive Word). I retested by copying and pasting it and it caused the same issue.
I then tried typing it manually into Mac OS Notes and copying and pasting from there and it happened again.
So it appears that somehow the apostrophe that is typed manually into Outlook seems to be different to one typed elsewhere.
